sass/project.sass

```diff
@@ -22,45 +26,118 @@
     +span-columns(6 of 12)
     +shift(3)
 
-
-.project__h1
+.project__title
   font-family: $adelleSans, sans-serif
   font-weight: 700
   text-transform: uppercase
   font-size: 4rem
   letter-spacing: 20px
 
-.project__subtitle
+.project__details,
+.project__subtitle,
+.project__content-title,
+.project__small-header,
+p
```

Comment:
Setting the element styles by using the `p` selector is gonna bite you in the ass later when you need to override it.

This composed rule might make more sense to be on the `.project` class or whatever the wrapper is called